src/OrbitGl/BatcherTest.cpp

```diff
@@ -237,4 +241,43 @@ TEST(Batcher, MultipleDrawCalls) {
   EXPECT_DEATH((void)batcher.GetUserData(id), "size");
 }
 
+bool LineEq(const Line& lhs, const Line& rhs) {
+  return lhs.start_point == rhs.start_point && lhs.end_point == rhs.end_point;
+}
+
+TEST(Batcher, TranslationsAreAutomaticallyAdded) {
+  MockBatcher batcher(BatcherId::kUi);
+  batcher.AddLine(Vec2(0.f, 0.f), Vec2(1.f, 1.f), 0.f, Color());
```

Comment:
Nit: Since you are adding the same line 4 times, could we have a constant or a small method there?

Reply:
Good point, I've added a method to add the line and verify the result.